graph/src/utils.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+
+def get_json_content(json_path):
+    """Get json content.
+
+    Args:
+        json_path (str): path to json.
+
+    Returns:
+        (dict): json content.
```

Comment:
Do you know [this](https://docs.python.org/3/library/typing.html) btw.?

Reply:
si ;) enterprise +1